integration/otlp_ingestion_test.go

```diff
@@ -398,6 +412,151 @@ func testOTLPHistogramIngestion(t *testing.T, enableExplicitHistogramToNHCB bool
 	require.Equal(t, result.(model.Vector)[0].Histogram.Sum, model.FloatString(25))
 }
 
+// TestStartTimeHandling implements E2E test for OTEL start time to
```

Comment:
[Nit] It's not an acronym.
```suggestion
// TestStartTimeHandling implements E2E test for OTel start time to
```